insalan/cms/models.py

```diff
@@ -78,6 +79,25 @@ def save(self, *args, **kwargs):
                 "border-right:1px SOLID #000000",
                 self.content
             )
+
+            letters = string.ascii_uppercase
+            for i in letters:
+                if f">{i}<" in self.content:
+                    self.content = self.content.replace(
+                        f">{i}<", "><"
+                    )
+                else:
+                    break
+            number = 1
+            while True:
```

Comment:
This is begging to freeze an entire thread

Reply:
It is indeed synchronous but it should not take a lot of time (00.011 seconds on my computer for example). I should test this on the beta server.

You could also transfer this code to the frontend (as discussed here : https://github.com/InsaLan/backend-insalan.fr/pull/118#discussion_r1438911601), but since it's always the same and delays page loading, I thought it best to put it in the back.

Reply:
And just, semantic wise, isn't it just a for loop, but less readable?

Reply:
We can't know how many rows they are in the schedule so we can't loop with a "for". For the columns (see above), we could limit ourself to 26 (whole alphabet) because too many columns will break readability on the website but there is a vertical scrollbar so we can expand it in height a lot more.
In python (unless I'm not aware of it), there's no way to write for loops with unknown break condition from the start.

Reply:
We can't know how many rows they are in the schedule so we can't loop with a "for". For the columns (see above), we could limit ourself to 26 (whole alphabet) because too many columns will break readability on the website but there is a vertical scrollbar so we can expand it in height a lot more.
In python (unless I'm not aware of it), there's no way to write for loops with unknown break condition from the start.

Reply:
>    We can't know how many rows they are in the schedule so we can't loop with a "for"

Can't you split the payload with delimiters for the columns and then for-iterate over that? it'd make it less likely that a logic bug freezes the backend